### PR TITLE
[win32] fix building after split addon gui for skin and binary addons

### PR DESCRIPTION
**Pull request by MartijnKaijser**
@FernetMenta 
some spots that were missed after the guilib split

### CODE REVIEW

**Comment by FernetMenta** [comment]
thanks! I hope we can get Kodi build on cmake too and get rid of those ugly VS and xcode projects

**Comment by MartijnKaijser** [comment]
hmm still something missing.

File: lib/addons/library.kodi.guilib/project/VS2010Express/post-build.ps1 [hunk 1]
```diff
@@ -3,7 +3,7 @@ param (
 )
 
 $LIB_INTERFACE = "$ProjectDir\..\..\..\..\..\addons\library.kodi.guilib\libKODI_guilib.h"
-$GENERATED_ADDON_GUILIB = "$ProjectDir\..\..\..\..\..\addons\kodi.guilib\addon.xml"
+$GENERATED_ADDON_GUILIB = "$ProjectDir\..\..\..\..\..\addons\library.kodi.guilib\addon.xml"
```
Comment (FernetMenta) [inline lib/addons/library.kodi.guilib/project/VS2010Express/post-build.ps1]: this is wrong

Comment (MartijnKaijser) [inline lib/addons/library.kodi.guilib/project/VS2010Express/post-build.ps1]: yes i see i went the wrong direction in this PR.
thought it needed to end up in library.kodi.guilib but needs to be kodi.guilib

**Pull request merged**